spec/helpers/application_helper/buttons/middleware_standalone_server_action_spec.rb

```diff
@@ -1,31 +1,50 @@
 describe ApplicationHelper::Button::MiddlewareStandaloneServerAction do
-  let(:record) { double("MiddlewareServer") }
-  subject(:action) { described_class.new(setup_view_context_with_sandbox({}), {}, {'record' => record}, {}) }
+  let(:record) { FactoryGirl.create(:middleware_server) }
+  let(:klass) { record }
```

Comment:
Different name please?

`klass` sounds like this should be `record.class`, not just `record`.